### PR TITLE
Marines doing defibs or manning mounted weapons cannot be pushed by other humans.

### PR DESCRIPTION
**Pull request by hyper2snyper**
## About The Pull Request

Title

## Why It's Good For The Game

SERIOUSLY STOP FUCKING PUSHING ME!

## Changelog
:cl:
qol: Marines doing defibs or manning mounted weapons cannot be pushed by other humans.
/:cl:

### CODE REVIEW

**Comment by Losenis** [comment]
PLEASE add surgery steps to this also

**Comment by Lumipharon** [comment]
Muh immersions.

Honestly I don't think this is warranted. Just stand next to a wall out of the way.
I don't want to have to stop wielding my gun to shove a medic that's decided to block a corridor while frontline defibbing.

**Comment by novaepee** [comment]
> Muh immersions.
> 
> Honestly I don't think this is warranted. Just stand next to a wall out of the way. I don't want to have to stop wielding my gun to shove a medic that's decided to block a corridor while frontline defibbing.

If there a few things in CM that I want in TGMC, it's marines not pushing corpsmen using defib on a red marine.

File: code/modules/mob/living/living.dm [hunk 1]
```diff
@@ -382,6 +382,9 @@
 		if(!(L.status_flags & CANPUSH))
 			return
 
+		if(HAS_TRAIT(L, TRAIT_NO_PUSHING_BY_HUMAN) && istype(src, /mob/living/carbon/human))
```
Comment (BraveMole) [inline code/modules/mob/living/living.dm:385]: is type src. Beeeeh.

And putting something that will almost never happen in bump, that happens all the time, is so meh as well.

Find another solution that does not require to meddle with a proc like that

**Review by BraveMole — CHANGES_REQUESTED** [review]

File: code/modules/mob/living/living.dm (hunk 1 above)
Comment (TiviPlus) [reply code/modules/mob/living/living.dm:385]: i dont see why setting anchored wouldnt work

**Pull request closed without merging**